serveradmin/servershell/templates/servershell/index.html

```diff
@@ -105,9 +105,9 @@
                 <div class="card">
                     <div class="card-header">
                         <b>Attributes</b>
-                        <button class="btn btn-link" type="button" data-toggle="collapse" data-target="#accordion-attributes-body"></button>
+                        <button class="btn btn-link collapsed" type="button" data-toggle="collapse" aria-expanded="false" id="attributesCollapseButton" data-target="#accordion-attributes-body" onclick="servershell.save_hidden_attrlist(this)"></button>
```

Comment:
When I look through the html lines here, I see snake_casing being used a lot for ids and other stuff. So, we should stick with it and not adding another casing:
```suggestion
                        <button class="btn btn-link collapsed" type="button" data-toggle="collapse" aria-expanded="false" id="attributes_collapse_button" data-target="#accordion-attributes-body" onclick="servershell.save_hidden_attrlist(this)"></button>
```
I would say, normally one use kebab-casing in html lines.

Reply:
Realized we don't even need to give ID here anymore.

> use kebab-casing

I see what you did there :smirk: